src/storage/movie.ts

```diff
@@ -0,0 +1,410 @@
+            "Country": "UK",
+            "Awards": "N/A",
+            "Poster": "https://m.media-amazon.com/images/M/MV5BNDczMzBkYTktMDAwYi00MjY0LTliODQtYWQ3ZjU2ZmVmOGRhL2ltYWdlL2ltYWdlXkEyXkFqcGdeQXVyNzcxNTkyMQ@@._V1_SX300.jpg",
+            "Ratings": [
+                {
+                    "Source": "Internet Movie Database",
+                    "Value": "8.1/10"
+                }
+            ],
+            "Metascore": "N/A",
+            "imdbRating": "8.1",
+            "imdbVotes": "8",
+            "imdbID": "tt2507642",
+            "Type": "movie",
+            "DVD": "N/A",
+            "BoxOffice": "N/A",
+            "Production": "N/A",
+            "Website": "N/A",
+            "Response": "True",
+            "comment": "Dont Like this movie",
+            "personalScore": 2.3,
+            "id": "7e52eda9-4fa9-4ce1-98f7-b37b985ed3bb"
+        },
+        "3cb502ec-3b11-47bd-a403-3f78d3f89492": {
+            "Title": "Richard Jewell",
+            "Year": "2019",
+            "Rated": "R",
+            "Released": "13 Dec 2019",
+            "Runtime": "131 min",
+            "Genre": "Biography, Crime, Drama",
+            "Director": "Clint Eastwood",
+            "Writer": "Billy Ray, Marie Brenner, Kent Alexander",
+            "Actors": "Paul Walter Hauser, Sam Rockwell, Brandon Stanley",
+            "Plot": "Security guard Richard Jewell is an instant hero after foiling a bomb attack at the 1996 Atlanta Olympics, but his life becomes a nightmare when the FBI leaks to the media that he is a suspect in the case.",
+            "Language": "English",
+            "Country": "United States",
+            "Awards": "Nominated for 1 Oscar. 7 wins & 20 nominations total",
+            "Poster": "https://m.media-amazon.com/images/M/MV5BOTFlODg1MTEtZTJhOC00OTY1LWE0YzctZjRlODdkYWY5ZDM4XkEyXkFqcGdeQXVyNjU1NzU3MzE@._V1_SX300.jpg",
+            "Ratings": [
+                {
+                    "Source": "Internet Movie Database",
+                    "Value": "7.5/10"
+                },
+                {
+                    "Source": "Rotten Tomatoes",
+                    "Value": "77%"
+                },
+                {
+                    "Source": "Metacritic",
+                    "Value": "68/100"
+                }
+            ],
+            "Metascore": "68",
+            "imdbRating": "7.5",
+            "imdbVotes": "75,836",
+            "imdbID": "tt3513548",
+            "Type": "movie",
+            "DVD": "10 Dec 2019",
+            "BoxOffice": "$22,345,542",
+            "Production": "N/A",
+            "Website": "N/A",
+            "Response": "True",
+            "comment": "Dont Like this movie",
+            "personalScore": 2.3,
+            "id": "3cb502ec-3b11-47bd-a403-3f78d3f89492"
+        }
+    },
+    total: 0
+};
+
+const get = (id: string) => {
+    return moviesList.movies[id];
+};
+
+const getAll = (sortOrder?: any, sortBy?: any, limit?: any, page?: any): Movie[] => {
```

Comment:
Better to don't use any and provide default values
`const getAll = (limit = 10, page = 1, ...)`